test/assets/messages/CalculationMessages.scala

```diff
@@ -17,17 +17,41 @@
 package assets.messages
 
 object CalculationMessages {
```

Comment:
Can the suffix 'Add' used in the vals below be refractory to 'Additional' please.